custom/guinea_backup/tasks.py

```diff
@@ -0,0 +1,42 @@
+from datetime import date
+from celery.schedules import crontab
+from celery.task import periodic_task
+from django.core.management import call_command
+from corehq.util.soft_assert import soft_assert
+from .models import BackupRecord
+
+import settings
+
+GUINEA_CONTACT_TRACING_DOMAIN = 'guinea_contact_tracing'
+GUINEA_CONTACT_TRACING_DATABASE = 'guineact-backup'
+
+
+@periodic_task(run_every=crontab(hour="1", day_of_week="7"), queue=settings.CELERY_PERIODIC_QUEUE)
+def copy_data_to_backup():
+    # https://<your_username>:<your_password>@commcarehq.cloudant.com/commcarehq
+    prod_couchdb_connection = 'https://{username}:{password}@commcarehq.cloudant.com/{database}'.format(
+        username=settings.COUCH_USERNAME,
+        password=settings.COUCH_PASSWORD,
+        database=settings.COUCH_DATABASE_NAME,
+    )
+    guinea_couchdb_connection = 'https://{username}:{password}@commcarehq.cloudant.com/{database}'.format(
+        username=settings.COUCH_USERNAME,
+        password=settings.COUCH_PASSWORD,
+        database=GUINEA_CONTACT_TRACING_DATABASE,
+    )
+    last_update = BackupRecord.objects.order_by('last_update')[0]
+
+    call_command('copy_domain',
```

Comment:
My big worry with it would be that it could create duplicate documents. This should fail pretty gracefully in that it will run the update the next week.  For this case I could add a try, catch that emails me on failure.  What else do you have in mind that could go wrong?